test/Java9andUp/src_latest/org/openj9/test/java9AttachAPI/VmIdGetter.java

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2015, 2017 IBM Corp. and others
```

Comment:
Can you explain the difference between src_latest and src_current?  I'm not aware of why there are two...

Reply:
This is a kludge to handle the API differences between the two JCL versions in the builds.